src/main/java/pro/belbix/tgnotifier/tg/InlineButton.java

```diff
@@ -0,0 +1,14 @@
+package pro.belbix.tgnotifier.tg;
+
+import lombok.Data;
+
+@Data
+public class InlineButton {
+    private String text = null;
```

Comment:
setup null not necessary, it is the default value for objects
make variables `final` if you fill it up only In constructor